code/game/machinery/computer/aifixer.dm

```diff
@@ -130,7 +125,7 @@
 		if(stat & (NOPOWER|BROKEN))
 			user << "[src] is offline and cannot take an AI at this time!"
 			return
-		AI.loc = src
+		AI.forceMove(src)

```

Comment:
to properly update the AI's perspective; otherwise the AI's eye would still follow the intellicard he was in previously.